SPECS/python3/python3.spec

```
@@ -1,19 +1,21 @@
 %global openssl_flags -DOPENSSL_NO_SSL3 -DOPENSSL_NO_SSL2 -DOPENSSL_NO_COMP
 %global __brp_python_bytecompile %{nil}
+%global majmin 3.9
```

Might be easier for future updates:

```suggestion
%global majmin %(echo %{version} | cut -d. -f1-2)
```

---

Won't fix- I did try this early on, but surprisingly it failed to cut the version down to maj.min! Possibly a future issue to fix with regards to toolchain builds.